Dockerfile

```diff
@@ -1,12 +1,16 @@
 FROM alpine:3.12
 
 ARG KUBE_VERSION="v1.16.10"
+ENV AWS_DEFAULT_REGION us-west-1
```

Comment:
informational: why `us-west-1` and not `us-west-2` ?

Reply:
No reason, making it so